hudi-common/src/main/java/org/apache/hudi/common/util/ClusteringUtils.java

```diff
@@ -69,7 +69,7 @@ public class ClusteringUtils {
   public static Stream<Pair<HoodieInstant, HoodieClusteringPlan>> getAllPendingClusteringPlans(
       HoodieTableMetaClient metaClient) {
     List<HoodieInstant> pendingReplaceInstants =
-        metaClient.getActiveTimeline().filterPendingReplaceTimeline().getInstants();
+        metaClient.reloadActiveTimeline().filterPendingReplaceTimeline().getInstants();
```

Comment:
It looks like all the invokers already have the refreshed timeline.

Reply:
multiple writer not do this，and heartbeat not set long engouh could cause it.All the next job would failed.